onnxruntime/contrib_ops/contrib_defs.h

```diff
@@ -3,8 +3,7 @@
 
 #pragma once
 
-#include "core/framework/op_kernel.h"
-#include "core/framework/kernel_registry.h"
+#include "onnx/defs/schema.h"
```

Comment:
replace this line with:
```c
#include "onnxruntime/core/graph/onnx_protobuf.h"
```

It will solve your current build failure.

Reply:
thank you very much! Changming.